### ISSUE TITLE
False positive ImplicitEnumerateViolation on range with step

### ISSUE THREAD

**Issue by AlwxSin** [wemake-services/wemake-python-styleguide#1742]
# Bug report

## What's wrong
A simple function that splits a list into chunks 
```python
def chunks(lst, amount):
    for index in range(0, len(lst), amount):
        yield lst[index:index + amount]
```
will raise `ImplicitEnumerateViolation`. 

Documentation suggests using `enumerate(lst)` instead of `range(0, len(list))` but there is no `step`

## How it should be
`ImplicitEnumerateViolation` should not raise if `step > 0` in range arguments.

**Comment by sobolevn** [comment]
@AlwxSin fix is welcome!

**Comment by AlwxSin** [comment]
@sobolevn yeah. I'll try to fix it in a week or two if no one else wouldn't take it.
